project-base/storefront/components/Basic/ModalGallery/ModalGallery.tsx

```diff
@@ -5,9 +5,12 @@ import { TypeFileFragment } from 'graphql/requests/files/fragments/FileFragment.
 import { TypeImageFragment } from 'graphql/requests/images/fragments/ImageFragment.generated';
 import { TypeVideoTokenFragment } from 'graphql/requests/products/fragments/VideoTokenFragment.generated';
 import useTranslation from 'next-translate/useTranslation';
+import NextImage from 'next/image';
```

Comment:
We should use `Image` from `components/Basic/Image/Image`. Please try to make these changes as part of this mentioned component.